community/labels.md

```diff
@@ -14,7 +14,7 @@
 ## Intro
 
 This file was auto generated by the [label_sync](https://git.k8s.io/test-infra/label_sync/) tool,
-based on the [labels.yaml](https://github.com/thoth-station/core/blob/master/labels.yaml) that it uses to
```

Comment:
This is an auto-generated file (see the `label-sync` target in the `Makefile`)

The changes should (also) be made in `labels.md.tmpl` so that future updates don't undo the change here

Reply:
Update in that file as well.
Thank you for pointing this out.